cookbook/asset/custom_version_strategy.rst

```diff
@@ -0,0 +1,264 @@
+.. index::
+   single: Asset; Custom Version Strategy
+
+How to use a Custom Version Strategy for Assets
+===============================================
+
+Symfony by default does not perform asset versioning. You can specify the
+:ref:`version <reference-framework-assets-version>` and
+:ref:`version_format <reference-framework-assets-version-format>` configuration
+options to add a simple version to all assets (or a specific set of assets
+grouped as a :ref:`package <reference-framework-assets-packages>`).
+
+.. versionadded:: 2.7
+    The Asset component was introduced in Symfony 2.7.
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # app/config/config.yml
+        framework:
+            assets:
+                version: "20150530"
+                version_format: "%%s?version=%%s"
+
+    .. code-block:: xml
+
+        <!-- app/config/config.xml -->
+        <?xml version="1.0" encoding="UTF-8" ?>
+        <container xmlns="http://symfony.com/schema/dic/services"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xmlns:framework="http://symfony.com/schema/dic/symfony"
+            xsi:schemaLocation="http://symfony.com/schema/dic/services
+                http://symfony.com/schema/dic/services/services-1.0.xsd
+                http://symfony.com/schema/dic/symfony
+                http://symfony.com/schema/dic/symfony/symfony-1.0.xsd">
+
+            <framework:config>
+                <framework:assets
+                    version="20150530"
+                    version-format="%%s?version=%%s" />
```

Comment:
Can you please move the closing `/>` to next line with the same indentation as the opening angle bracket?

It should look like this then:

```xml
<framework:assets
    version="20150530"
    version-format="%%s?version=%%s"
/>
```